tableauserverclient/models/connection_credentials.py

```diff
@@ -0,0 +1,5 @@
+class ConnectionCredentials(object):
```

Comment:
Can you add a docstring on this class that tells users they should delete this object when they're done using it?